samples/application-conf/zCEE3.properties

```diff
@@ -2,24 +2,26 @@
 
 #
 # Comma separated list of required build properties for zCEE3.groovy
-zCEE_requiredBuildProperties=zCEE_alwaysRebuildWAR, zCEE_shellEnvironment,\
-                                zCEE_gradlePath, zCEE_gradle_JAVA_OPTS
+zcee3_requiredBuildProperties=zcee3_shellEnvironment, zcee3_gradlePath, zcee3_gradle_JAVA_OPTS
 
 
 #
 # When set to true, always rebuild the WAR file, even if one is found in the build folder of the z/OS Connect project folder.
 # When set to false, will pick the existing WAR file if found or will build it if no WAR file is found
-# zCEE_alwaysRebuildWAR=true
+# When not specified, the default value is set to true
+#zcee3_alwaysRebuildWAR=
```

Comment:
@M-DLB Still believe, we should not offer the property at all, because someone could just build it somewhere (locally), then inject the WAR into the repo and nobody knows what the input to this WAR was.

Reply:
Fine, and I tend to agree! Completely removed :)